test/rendition.content.test.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-env mocha */
+/* eslint mocha/no-mocha-arrows: "off" */
+
+'use strict';
+
+const assert = require('assert');
+const Rendition = require('../lib/rendition.js');
+
+// WARNING: filesystem is not mocked here, so content-type identification can be tested with known files.
+
+describe("rendition.js - content types", () => {
```

Comment:
Do these tests depend on the existence of the `file` command?

I think we should mock it, and be able to test both file and file-type fallback cases.